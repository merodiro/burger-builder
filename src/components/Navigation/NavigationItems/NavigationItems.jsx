import React from 'react'

import classes from './NavigationItems.module.css'
import NavigationItem from './NavigationItem/NavigationItem'

const NavigationItems = () => (
  <nav className={classes.NavigationItems}>
    <ul>
      <NavigationItem link="/" active>
        Burger Builder
      </NavigationItem>
      <NavigationItem link="/">Checkout</NavigationItem>
    </ul>
  </nav>
)

export default NavigationItems
