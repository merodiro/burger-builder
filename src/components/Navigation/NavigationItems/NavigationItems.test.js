import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'

import NavigationItems from './NavigationItems'

describe('<NavigationItems>', () => {
  it('renders the same output', () => {
    const component = shallow(<NavigationItems />)
    expect(component).toMatchSnapshot()
  })
})
