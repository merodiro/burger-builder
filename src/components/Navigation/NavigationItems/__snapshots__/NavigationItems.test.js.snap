// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationItems> renders the same output 1`] = `
<nav
  className="NavigationItems"
>
  <ul>
    <NavigationItem
      active={true}
      link="/"
    >
      Burger Builder
    </NavigationItem>
    <NavigationItem
      link="/"
    >
      Checkout
    </NavigationItem>
  </ul>
</nav>
`;
